ultralytics/solutions/object_counter.py

```diff
@@ -263,6 +301,31 @@ def start_counting(self, im0, tracks):
             self.display_frames()
         return self.im0
 
+    def greet(self, name):
+        print("Inheritance success")
+        print(name)
+
+    """
+        The function inherits the dictionaries
+        Incoming list, outgoing list, and two-way(both incoming and outgoing) list
+        
+        Incoming and outgoing lists sample: {'person': 0, 'cell phone': 0}
+        two_way sample: {'person': {'out': 0, 'in': 0}, 'cell phone': {'out': 0, 'in': 0}}
+    """
+
+    def inherit_dictionary(self, object_list):
```

Comment:
The argument `object_list` may cause users to think a `list` object should be passed, although it appears it probably _should_ be a `dict` object. Consider type hints, renaming, and/or type checking/assertion.